src/global_scheduler/global_scheduler_algorithm.cc

```diff
@@ -118,6 +118,10 @@ bool handle_task_waiting(GlobalSchedulerState *state,
                          GlobalSchedulerPolicyState *policy_state,
                          Task *task) {
   TaskSpec *task_spec = Task_task_execution_spec(task)->Spec();
+  TaskExecutionSpec *task_exe_spec = Task_task_execution_spec(task);
+
+  LOG_INFO("[GS][handle_task_waiting] task was spilled over %d times\n",
+           task_exe_spec->getSpillbackCount());
```

Comment:
Before merging please remove or downgrade to `LOG_DEBUG`

Reply:
all logging is temporary and was planned for removal as the last step of the WIP.